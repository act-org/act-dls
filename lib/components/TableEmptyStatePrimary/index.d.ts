/**
 * @prettier
 */
import * as React from 'react';
import { Props as EmptyStatePrimaryProps } from '~/components/EmptyStatePrimary';
export declare type Props = EmptyStatePrimaryProps;
declare const TableEmptyStatePrimary: React.FC<Props>;
export default TableEmptyStatePrimary;
//# sourceMappingURL=index.d.ts.map