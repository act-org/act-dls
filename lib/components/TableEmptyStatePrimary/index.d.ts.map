{"version":3,"file":"index.d.ts","sourceRoot":"","sources":["../../../src/components/TableEmptyStatePrimary/index.tsx"],"names":[],"mappings":"AAAA;;GAEG;AAEH,OAAO,KAAK,KAAK,MAAM,OAAO,CAAC;AAG/B,OAA0B,EACxB,KAAK,IAAI,sBAAsB,EAChC,MAAM,gCAAgC,CAAC;AAExC,oBAAY,KAAK,GAAG,sBAAsB,CAAC;AAE3C,QAAA,MAAM,sBAAsB,EAAE,KAAK,CAAC,EAAE,CAAC,KAAK,CAkB3C,CAAC;AAEF,eAAe,sBAAsB,CAAC"}