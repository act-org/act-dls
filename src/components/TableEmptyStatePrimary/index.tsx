/**
 * @prettier
 */

import * as React from 'react';

import DIMS from '~/constants/dims';
import EmptyStatePrimary, {
  Props as EmptyStatePrimaryProps,
} from '~/components/EmptyStatePrimary';

export type Props = EmptyStatePrimaryProps;

const TableEmptyStatePrimary: React.FC<Props> = (
  props: Props,
): React.ReactElement<any> => (
  <>
    <style jsx>
      {`
        .container {
          padding: ${DIMS.LAYOUT_PADDING * 2.5}px ${DIMS.LAYOUT_PADDING}px
            ${DIMS.LAYOUT_PADDING}px ${DIMS.LAYOUT_PADDING}px;
          width: 100%;
        }
      `}
    </style>

    <div className="container">
      <EmptyStatePrimary description="No Results Found" {...props} />
    </div>
  </>
);

export default TableEmptyStatePrimary;
