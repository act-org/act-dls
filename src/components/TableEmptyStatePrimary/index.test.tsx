/**
 * @prettier
 */

import * as React from 'react';
import renderer from 'react-test-renderer';

import TableEmptyStatePrimary from '.';

describe('TableEmptyStatePrimary', () => {
  it('matches the snapshot', () => {
    const tree = renderer
      .create(<TableEmptyStatePrimary description="No Things Found" />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
