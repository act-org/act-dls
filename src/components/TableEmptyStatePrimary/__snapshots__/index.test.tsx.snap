// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableEmptyStatePrimary matches the snapshot 1`] = `
Array [
  <style
    jsx={true}
  >
    
        .container {
          padding: 30px 12px
            12px 12px;
          width: 100%;
        }
      
  </style>,
  <div
    className="container"
  >
    <style
      jsx={true}
    >
      
          .container {
            align-items: center;
            display: flex;
            flex: 1;
            flex-direction: column;
          }
        
    </style>
    <div
      className="container"
    >
      <p
        className="MuiTypography-root makeStyles-descriptionRoot-1 makeStyles-descriptionRootWithoutTitle-3 MuiTypography-body1 MuiTypography-alignCenter"
      >
        No Things Found
      </p>
    </div>
  </div>,
]
`;
